lib/i18n_override.rb

```diff
@@ -0,0 +1,93 @@
+require 'i18n'
+
+I18n.module_eval do
+  class << self
+    def translate_with_markup(*args)
+      i18n_text = normal_translate(*args)
+      return i18n_text unless FeatureManagement.enable_dev_mode? && i18n_text.is_a?(String)
+
+      key = args.first.to_s
+      rtn = i18n_text + additional_markup(key)
+
+      rtn.html_safe
+    end
+
+    private
+
+    def additional_markup(key)
```

Comment:
how about naming this `add_dev_mode_translation_markup` ? When I first read through this file, I thought it was some crazy way of adding HTML to translations. Which it kind of is, but for a specific reason 🌈